R-package/R/utils.R

```diff
@@ -19,6 +19,36 @@ lgb.encode.char <- function(arr, len) {
 
 }
 
+lgb.last_error <- function() {
```

Comment:
I like this change! I'm surprised we didn't have a function like this before, actually.